env/fs_posix.cc

```diff
@@ -766,9 +766,9 @@ class PosixFileSystem : public FileSystem {
     // closed, all locks the process holds for that *file* are released
     const auto it_success = locked_files.insert({fname, lhi});
     if (it_success.second == false) {
+      LockHoldingInfo& prev_info = it_success.first->second;
```

Comment:
is this needed?

Reply:
This is from #7721 
We also faced with this problem. I can delete this from here, but could you please review that PR?